### PR TITLE
Kinetic devel update from indigo

### PR DESCRIPTION
**Pull request by v-lopez**
Catching up with the fixes in indigo-devel.

We can either accept this PR, or we can rebase kinetic-devel on top of indigo-devel.

### CODE REVIEW

File: aruco/src/aruco/marker.cpp [hunk 1]
```diff
@@ -287,7 +287,7 @@ namespace aruco {
     //rotate the X axis so that Y is perpendicular to the marker plane
     if (setYPerpendicular) rotateXAxis(Rvec);
     ssize=markerSizeMeters;
-    cout<<(*this)<<endl;
+    //cout<<(*this)<<endl;
```
Comment (bmagyar) [inline aruco/src/aruco/marker.cpp]: this could be removed altogether

**Review by bmagyar — CHANGES_REQUESTED** [review]

**Comment by bmagyar** [comment]
Other than that, all good.
About rebasing or merging this over: It's up to you how clean you wish to keep the history. This amount of stuff is probably messy both ways.

File: aruco/src/aruco/marker.cpp (hunk 1 above)
Comment (v-lopez) [reply aruco/src/aruco/marker.cpp]: You are right, I'll get to it.

**Comment by v-lopez** [comment]
I'll probably rebase this last time.

Then I'll create a release for 1.0.0 for kinetic-devel, and from that point future changes from indigo-devel shall be ported via cherry-pick without breaking the history.

**Comment by v-lopez** [comment]
@bmagyar I'm closing this and rebasing kinetic-devel on top of indigo-devel.

**Comment by jdlangs** [comment]
@v-lopez I don't think `kinetic-devel` should ever be rebased because that will invalidate everyone's local clone.

**Comment by bmagyar** [comment]
The majority of the code-base should be changed to use OpenCV3 which is available with Kinetic. That will render any local changes invalid invalid anyways. 

Also, if you've got changes to be  contributed, feel free to bring them forward!

**Comment by jdlangs** [comment]
Sorry, I wasn't clear. Breaking changes on the branch are fine (and welcome). But rebasing it rewrites it's history so our local copies effectively become separate branches and we would have to all delete or rebase our stuff to sync it with this remote.

**Pull request closed without merging**